visualizer/index.html

```diff
@@ -467,6 +470,7 @@
 
   <div id="options">
     <ul>
+      <li><input type="checkbox" name="showExampleGenerator"><label>Show Example Generator</label></li>
```

Comment:
Is this for the name or the label? The only thing that's confusing about 'generate examples' is that checking the box doesn't actually generate examples. It just shows the UI